cmd/init/config.go

```diff
@@ -11,48 +11,56 @@ import (
 )
 
 func generatehcl2NixConf(pt langdetect.ProjectType, pd *langdetect.ProjectDetails) (hcl2nix.Config, error) {
+	fmt.Println("IN GENERATE HCL NIX CONFIG -------------------------------------------")
+	fmt.Println("NAME: ", pd.Name)
 	switch pt {
 	case langdetect.GoModule:
 		return genGoModuleConf(pd), nil
 	case langdetect.PythonPoetry:
-		return genPythonPoetryConf(), nil
+		return genPythonPoetryConf(pd), nil
 	case langdetect.RustCargo:
-		config, err := genRustCargoConf()
+		config, err := genRustCargoConf(pd)
 		if err != nil {
 			return hcl2nix.Config{}, err
 		}
 		return config, nil
 	case langdetect.JsNpm:
-		config, err := genJsNpmConf()
+		config, err := genJsNpmConf(pd)
 		if err != nil {
 			return hcl2nix.Config{}, err
 		}
 		return config, nil
 	default:
-		return generateEmptyConf(), nil
+		return generateEmptyConf(pd), nil
 	}
 }
 
-func generateEmptyConf() hcl2nix.Config {
+func generateEmptyConf(pd *langdetect.ProjectDetails) hcl2nix.Config {
+	if pd.Name == "" {
+		pd.Name = "expl"
+	}
 	return hcl2nix.Config{
 		Packages: hcl2nix.Packages{
 			Development: []string{""},
 			Runtime:     []string{"cacert@3.95"},
 		},
 		OCIArtifact: []hcl2nix.OCIArtifact{
 			{
-				Artifact: "pkgs",
-				Name:     "bsf-base-image",
-				Cmd:      []string{},
-				Entrypoint: []string{},
-				EnvVars:    []string{},
-				ExposedPorts: []string{},
+				Artifact:      "pkgs",
+				Name:          pd.Name,
```

Comment:
this is the image name. It should be something like `ttl.sh/repo/app`. 
Also, let's make it a required field i.e user should specify the name.